challenge_restaurant/README.md

```diff
@@ -3,28 +3,12 @@
 Responsible: Rein
```

Comment:
Maintainer is Ramon, please also remove year.

Reply:
Fixed in 9ecad01e3839e2c035db397853fbd39948c1c2ba